cluster-autoscaler/processors/status/empty_cluster_processor.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package status
```

Comment:
I don't think this processor should go into status package. Processors in this package are generally called after a piece of logic (scaleup/scaledown/RunOnce) is ~done and passed detailed information about what CA has done. The most obvious use of those is monitoring / status reporting, but they're generally executed too late to influence CA actions.
This one doesn't really have any of those characteristics, so I'd lean towards putting it in its own package.

Reply:
sounds good.